cloudconfig.tpl

```diff
@@ -763,16 +763,56 @@ write_files:
       echo "$${appDirs}" | xargs -n 1 cp -v -f /home/wowza/Application.xml
   - owner: wowza:wowza
     permissions: 0775
-    path: /home/wowza/check-cert.sh
+    path: /home/wowza/check-file-size.sh
     content: |
         #!/bin/bash
 
         # Project
         project="CVP"
 
-        # Set Dynatrace Details.
-        dynatrace_token="${dynatrace_token}"
```

Comment:
why have you removed these vars? this will break the scripts.

Reply:
missing by mistake. Added back in now